spread.yaml

```diff
@@ -0,0 +1,28 @@
+project: classic-snap
+
+path: /home/spread
+
+backends:
+  qemu:
+    systems:
+      - ubuntu-16.04-64:
+          username: ubuntu
+          password: ubuntu
+      - ubuntu-18.04-64:
+          username: ubuntu
+          password: ubuntu
+
+suites:
+  tests/main/:
+    summary: Integration test for the classic snap
+    prepare: |
+      echo "Prepare image"
```

Comment:
Suggestion for a follow-up: move this to a `prepare-suite.sh` script.

Reply:
Yeah, makes sense, once I start adding 14.04 I will have to have more conditionals (14.04 needs extra handling). Right now I feel its ok because its still pretty trivial.